src/templates/homepage/HeroSection.jsx

```diff
@@ -37,7 +42,7 @@ const HeroButton = ({ button }) => (
 
 const HeroDropdownElem = ({ title }) => (
   <div className={editorStyles["bp-dropdown-item"]}>
-    <h5>{title}</h5>
+    <h5>{title || DROPDOWN_ELEMENT_SECTION.title}</h5>
   </div>
```

Comment:
Is there a reason why we pass in hero section this way instead of like the other preview sections? (destructuring existing object so that only non-blank params will overwrite)

Reply:
don't the other preview adopt this pattern also?

Reply:
oh cos this way was less work - i'm open to keeping it consistent tbh

Reply:
this is abit annoying to do here cos we have to do a type cast + overwrite a child array. lmk if this is a blocker for u

Reply:
I'm okay with leaving it as is if it's annoying to implement!